types/zumly/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for zumly 0.9
+// Project: https://zumly.org
+// Definitions by: edwinork <https://github.com/edwinork>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+    /**
+     * Zumly powers your apps with a zoomable user interface (ZUI) taste.
+     */
+    class Zumly {
```

Comment:
When used as a script in the browser, the global name is `zumly` not `Zumly`: https://unpkg.com/browse/zumly@0.9.9/dist/zumly.umd.js (line 9)

Reply:
I verified this by modifying the [codepen linked on the site](https://codepen.io/zumly/pen/gOPQovd) to use `import "https://unpkg.com/zumly@0.9.9/dist/zumly.umd.js";` and had to modify the call to `new Zumly` to use `new zumly` for the codepen to work when loaded as a script.

Reply:
Thanks for the catch! Fixed.